src/index.spec.ts

```diff
@@ -4,9 +4,195 @@ import {
   removeDuplicatesFromArray,
   createRandomProduct,
   getStarWarsPlanets,
-  createProduct 
+  createProduct,
 } from './index';
 
-test('Testing time!', () => {
-  console.log('welcome');
+describe('isInteger tests', () => {
```

Comment:
for more readability when you define a describe block only put the name of the function to be tested, for this case should be `describe('isInteger', () => {`